### PR TITLE
Fixes #138 redirect after login

### LINKED ISSUE #138: Support remembering originating URL when redirecting to login
**Note: for support questions, please use one of these channels:** [Chat: AngularClass.slack](http://angularclass.com/member-join/) or [Twitter: @AngularClass](https://twitter.com/AngularClass)

* **I'm submitting a ...**
[ ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**

* **What is the current behavior?**

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via
https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5).

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**

- Angular version: 2.0.0-beta.X
- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: src/app/run/pipelines/pipelines-routing.module.ts [hunk 1]
```diff
@@ -1,17 +1,19 @@
-import { NgModule }  from '@angular/core';
+import { AuthGuard } from './../../shared/auth-guard.service';
```
Comment (joshuawilson) [inline src/app/run/pipelines/pipelines-routing.module.ts]: For consistency we have the imports done in the groups. First is Angular, then external modules, then internal. 

Sometimes I try to alphabetize them within their group too.

File: src/app/run/run-routing.module.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { AuthGuard } from './../shared/auth-guard.service';
```
Comment (joshuawilson) [inline src/app/run/run-routing.module.ts]: you are not using AuthGuard

File: src/app/shared/login.service.ts [hunk 3]
```diff
@@ -22,4 +30,18 @@ export class LoginService {
     window.location.href = this.githubUrl;
   }
 
+  public redirectAfterLogin() {
+    let redirectUrl = this.localStorage.get<string>(this.REDIRECT_URL_KEY);
```
Comment (joshuawilson) [inline src/app/shared/login.service.ts]: we should work out how to do this without local storage, maybe Dana can help with that.

**Comment by pmuir** [comment]
Why?

File: src/app/shared/login.service.ts (hunk 3 above)
Comment (pmuir) [inline src/app/shared/login.service.ts]: Out of interest, why?

**Pull request merged**